### PR TITLE
Add Client Names (#1363)

### PR DESCRIPTION
**Pull request by unlimish**
- Add 3 client names
  -  -UW110Q-
  uTorrent Web
  - -FW6830-
  FrostWire
  - -BI2300-
  BiglyBT (2.3.0.0)
- Edited ` libtransmission/clients.c`
- Add 3 else if condition in `tr_clientForId()` as `no_version()`

### CODE REVIEW

File: libtransmission/clients.c [hunk 1]
```diff
@@ -683,6 +683,18 @@ char* tr_clientForId(char* buf, size_t buflen, void const* id_in)
     {
         no_version(buf, buflen, "BitTyrant (Azureus Mod)");
     }
+    else if (strncmp(chid, "-UW110Q-", 8) == 0)
+    {
+        no_version(buf, buflen, "uTorrent Web");
+    }
+    else if (strncmp(chid, "-UW110Q-", 8) == 0)
```
Comment (ckerr) [inline libtransmission/clients.c:690]: Shouldn't this begin with `FW` rather than `UW`?

**Review by ckerr — COMMENTED** [review]
All three of these are hardcoding the version number into the peer-id parser. Is that intentional?

Check how the other flavors of uTorrent are parsed, e.g. uTorrent Embedded, uTorrent Mac:

```c
          else if (strncmp(chid + 1, "UM", 2) == 0)
          {
              tr_snprintf(buf, buflen, "\xc2\xb5Torrent Mac %d.%d.%d%s", strint(id + 3, 1), stri  nt(id + 4, 1), strint(id + 5, 1),
                  getMnemonicEnd(id[6]));
          }
          else if (strncmp(chid + 1, "UE", 2) == 0)
          {
              tr_snprintf(buf, buflen, "\xc2\xb5Torrent Embedded %d.%d.%d%s", strint(id + 3, 1),   strint(id + 4, 1),
                  strint(id + 5, 1), getMnemonicEnd(id[6]));
          }
```
I think you would want to do the same here.

**Comment by unlimish** [comment]
No, That isn't intentional. I'll fix basing on that example.

**Comment by unlimish** [comment]
> 
> 
> All three of these are hardcoding the version number into the peer-id parser. Is that intentional?
> 
> Check how the other flavors of uTorrent are parsed, e.g. uTorrent Embedded, uTorrent Mac:
> 
> ```c
>           else if (strncmp(chid + 1, "UM", 2) == 0)
>           {
>               tr_snprintf(buf, buflen, "\xc2\xb5Torrent Mac %d.%d.%d%s", strint(id + 3, 1), stri  nt(id + 4, 1), strint(id + 5, 1),
>                   getMnemonicEnd(id[6]));
>           }
>           else if (strncmp(chid + 1, "UE", 2) == 0)
>           {
>               tr_snprintf(buf, buflen, "\xc2\xb5Torrent Embedded %d.%d.%d%s", strint(id + 3, 1),   strint(id + 4, 1),
>                   strint(id + 5, 1), getMnemonicEnd(id[6]));
>           }
> ```
> 
> I think you would want to do the same here.

No, That isn't intentional. I'll fix basing on that example.

**Pull request closed without merging**